images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -502,6 +502,11 @@ function Get-SwigVersion {
     return "Swig $swigVersion"
 }
 
+function Get-BicepVersion {
+    $bicepVersion = Run-Command "bicep --version" | Take-Part -Part 3
+    return "Bicep $bicepVersion"
```

Comment:
Should it be `Bicep CLI` as in output?
![image](https://user-images.githubusercontent.com/48208649/124269168-34cbdf00-db43-11eb-973a-157d179958ae.png)

Reply:
We could add CLI. Do you think I should add it to the variable `$bicepVersion` and function name or in return statement?

Reply:
The return statement should be fine. Thanks!

Reply:
@MarcusFelling could you change the return statement to the following, please?
`return "Bicep CLI $bicepVersion"`

Reply:
Done